salt/search/config/etc-init-search-queue-watch.conf

```diff
@@ -0,0 +1,9 @@
+description "search-queue-watch Will listen to the SQS queue and pass items to gearman to import"
+respawn
+kill timeout 70
+setuid {{ pillar.elife.deploy_user.username }}
+instance $ID
+chdir /srv/search
+script
+    exec php bin/console queue:watch search--{{ pillar.elife.env }} -e {{ pillar.elife.env }}
```

Comment:
this is missing `$ID` at the end like at https://github.com/stephenwf/search-formula/blob/93d68a7294fa2bc59c21b95ecf419b9da056edb9/salt/search/config/etc-init-search-gearman-worker.conf#L8